src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -1,42 +1,54 @@
 package seedu.duke.parser;
 
-import seedu.duke.command.AddModuleCommand;
-import seedu.duke.command.Command;
-import seedu.duke.command.DeleteModuleCommand;
-import seedu.duke.command.ExitCommand;
-import seedu.duke.command.HelpCommand;
-import seedu.duke.command.IncompleteCommand;
-import seedu.duke.command.InvalidModuleCommand;
-import seedu.duke.command.UnknownCommand;
-import seedu.duke.command.ViewTimetableCommand;
-import seedu.duke.command.SelectSlotCommand;
-import seedu.duke.command.SearchModuleCommand;
+import seedu.duke.command.*;
```

Comment:
i think we should avoid wildcard imports

Reply:
Fixed